0_RootFS/Go/build_tarballs.jl

```diff
@@ -3,22 +3,24 @@ using BinaryBuilder
 include("../common.jl")
 
 name = "Go"
-version = v"1.13"
+version = v"1.16.3"
```

Comment:
I'll note that Go seems to treat a minor version change as a major release: https://golang.org/doc/devel/release.html#go1.16.

Reply:
The Go ecosystem has a wonderful versioning scheme :grimacing:

Reply:
It does seem though that 1.16 is supposed to be backwards compatible with all version 1 releases